nginx/Makefile

```
@@ -0,0 +1,14 @@
+COMPOSE_FILE=docker-compose.test.yml
+
+docker_build:
+	docker build --target test_nginx .
+
+start: docker_build
+	env BUILD_TARGET=cht-nginx docker compose -f ${COMPOSE_FILE} up --wait --wait-timeout 15 --build
+
+test: start
+	bats tests/compose.bats
+	$(MAKE) stop
+
+stop:
+	docker compose -f ${COMPOSE_FILE} stop	
```

Some comments about this:

1.  LOVE THIS!  So easy to run tests with a file that puts all the logic in one place
2. it seems excessively [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) to me.  Is there a good reason to have only the `test` call with this?
   ```suggestion
   COMPOSE_FILE=docker-compose.test.yml

   test: docker build --target test_nginx .
	   docker compose -f ${COMPOSE_FILE} up --wait  --build
	   bats tests/compose.bats
	   docker compose -f ${COMPOSE_FILE} stop
   ```
4. ` --wait-timeout 15` was [added only in Feb](https://github.com/docker/compose/pull/10276#pullrequestreview-1297333028) so I didn't have it on my `docker` version.  In the suggestion above I've removed it - is that OK?  Given [my read of what it does](https://github.com/docker/compose/issues/10269#issue-1580826714), it seems safe, but maybe not? Don't feel strongly here. 
3. I couldn't find out what setting this did, can you explain it:  `env BUILD_TARGET=cht-nginx` ? The tests performed the same with and without it, so maybe it's not needed? I've removed it in the suggestion above. Again, don't feel strongly here.

---

2 - `docker_build` can be removed as it's no longer needed since compose has `--build`. I'd prefer to either keep the other steps or renaming `docker-compose.test.yml` to `compose.yml` so the `-f` param isn't needed when starting locally
3 - It's safe to remove, It's just there to fail faster in case it were to hang indefinitely, which is highly unlikely and we can rely on github actions timeout there.
4 - old leftover from the compose file that I removed, will delete